lib/scsi_defs.h

```diff
@@ -195,6 +195,100 @@ enum class PageCode : uint8_t {
   kLogicalBlockProvisioningVpd = 0xb2
 };
 
+// SCSI Reference Manual Table 440
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+enum class MediumRotationRate : uint16_t {
+  kRotationNotReported = 0x0000,
+  kNonRotatingMedium = 0x0001,
+  kReserved = 0xFFFF,
+};
+
+// SCSI Reference Manual Table 441
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+enum class ProductType : uint8_t {
+  kNotIndicated = 0x00,
+  kCFast = 0x01,
+  kCompactFlash = 0x02,
+  kMemoryStick = 0x03,
+  kMultiMediaCard = 0x04,
+  kSecureDigitalCard = 0x05,
+  kXQD = 0x06,
+  kUniversalFlashStorage = 0x07
+};
+
+// SCSI Reference Manual Table 442
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+enum class Wacereq : uint8_t {
```

Comment:
nit: is it possible to add a comment that specifies what `wacereq` stands for. Similarly for `wabereq`

Reply:
yeah for sure, will add that. Also, what are your views on these two enums. They have the exact same content inside so I was wondering if we can re-use them somehow or should leave them as two separate enums.